### PR TITLE
Adding step to get the CA certificate

### PR DESCRIPTION
**Pull request by bergerhoffer**
@sjenning @sunilcio Can you please review this update?

Preview: http://file.rdu.redhat.com/~ahoffer/2019/dr-update/disaster_recovery/scenario-3-expired-certs.html

FYI @blomquisg

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@xltian if this passes QE, we will need to approve it for inclusion in the docs for GA.

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 1]
```diff
@@ -113,6 +113,12 @@ cat "$KUBECONFIG"
 
 .. Copy the `kubeconfig` file to all master hosts and move it to `/etc/kubernetes/kubeconfig`.
 
+.. Get the CA certificate used to validate connections from the API server.
++
+----
+# oc get configmap kube-apiserver-to-kubelet-client-ca -n openshift-kube-apiserver-operator --template='{{ index .data "ca-bundle.crt" }}' > /etc/kubernetes/ca.crt
+----
+
```
Comment (zhouying7780) [inline modules/dr-recover-expired-control-plane-certs.adoc]: According with @tnozicka  command  , we should distribute the `ca.crt` to other masters and nodes and then restart the kubelet service.

Comment (bergerhoffer) [reply modules/dr-recover-expired-control-plane-certs.adoc]: @sjenning Can you confirm? Do we need to add a step to copy this file to all other masters at /etc/kubernetes/ca.crt?

Comment (bergerhoffer) [reply modules/dr-recover-expired-control-plane-certs.adoc]: @zhouying7780 I've updated the doc to add a step to copy the file to the other master hosts.

Comment (zhouying7780) [reply modules/dr-recover-expired-control-plane-certs.adoc]: @bergerhoffer  During test I also copy the file to other nodes hosts too.

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 2]
```diff
@@ -113,6 +113,14 @@ cat "$KUBECONFIG"
...
+.. Copy the `ca.crt` file to all other master hosts and move it to `/etc/kubernetes/ca.crt`.
```
Comment (tnozicka) [inline modules/dr-recover-expired-control-plane-certs.adoc]: this should go to nodes as well

Comment (tnozicka) [reply modules/dr-recover-expired-control-plane-certs.adoc]: Maybe phrase it in lines of: "Copy `/etc/kubernetes/ca.crt` to all other masters and nodes."

**Comment by sunilcio** [comment]
Followed new updated steps and all looks good.

As tnozicka mentioned, we need to modify step 9.f to mention to copy `/etc/kubernetes/ca.crt` to all masters and nodes.

File: modules/dr-recover-expired-control-plane-certs.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/dr-recover-expired-control-plane-certs.adoc]: Updated.

**Comment by bergerhoffer** [comment]
Got the okay, merging!

**Pull request merged**